plugins/circleci-deploy/src/index.ts

```diff
@@ -28,6 +29,31 @@ const generateConfigWithServerlessOptions = (
   return generateConfigWithJob(jobOptions)
 }
 
+const getServerlessAdditionalFields = (logger: Logger): JobConfig => {
+  const serverlessOptions = getOptions('@dotcom-tool-kit/serverless')
+  const herokuOptions = getOptions('@dotcom-tool-kit/heroku')
+
+  if (serverlessOptions && herokuOptions) {
+    logger.warn(
+      'Tool Kit currently does not support managing Heroku and Serverless apps in the same project.'
+    )
+  }
+
+  if (!serverlessOptions?.awsAccountId || !serverlessOptions?.systemCode) {
+    return {}
+  }
+
+  return {
+    'aws-account-id': serverlessOptions.awsAccountId,
+    'system-code': serverlessOptions.systemCode
+  }
+}
+
+const CYPRESS_JOB_OPTIONS = {
```

Comment:
**nitpick**: I think this is actually unneeded deduplication – many of the other classes have these two properties set regardless of whether they use Cypress or not